lib/msal-common/test/auth/authority/AuthorityFactory.spec.ts

```diff
@@ -14,26 +16,58 @@ describe("AuthorityFactory.ts Class Unit Tests", () => {
             return null;
         }
     };
+
+    beforeEach(() => {
```

Comment:
what does this do for the test?

Reply:
Cleans up the static host list between tests so the B2C tests dont break the AAD tests.

Reply:
A commend would probably helpful here to explain that.

Reply:
Done